CHANGES/6894.bugfix

```diff
@@ -0,0 +1 @@
+Refactored token_authentication to be available in api-app.
```

Comment:
how about -  Refactored token_authentication that now happens in pulpcore-api app